docs/Documentation/Configuration.md

```diff
@@ -0,0 +1,80 @@
+## Configuration
+
+Before you start with the configuration process, please ensure that the extension is properly installed on a working CiviCRM and that you have access to a functioning mailbox that you can and want to use for your MailboxMailing setup.
+
+### Compatability
+
+The MailboxMailing extension is compatible with every IMAP or POP3 mailserver. You need access to the mailbox via webmail or you can use basically every modern mailing program such as Gmail, Outlook, Apple icloud mail, yahoo and many others. You can also use Maildir oder Localdir to use the extension without having to connect through a mailserver like Outlook or Gmail.
+
+### Set up the Mailbox Connection
+
+To set up the Mailbox Connection go to *Administer* - *Administration Console* - *MailboxMailing settings.* Here you can *add Mailbox settings.* You can set a names for the groups of settings and create multiple groups for different mailboxes.
```

Comment:
Let's find another term for what is described as "groups of settings", as "Group" is a specific CiviCRM entity that is used in another context and might be confusing.